core/components/atoms/input/Input.tsx

```diff
@@ -223,13 +223,15 @@ export const Input = React.forwardRef<HTMLInputElement, InputProps>((props, forw
   );
 
   return (
+    // eslint-disable-next-line
     <div
       data-test="DesignSystem-InputWrapper"
       className={classes}
       style={{ minWidth }}
       onClick={() => ref.current?.focus()}
-      role="presentation"
+      role="button"
```

Comment:
Please use the appropriate role based on component usage. Role here should be `textbox`.

Reply:
done